frontend/components/feedback-task/SearchBar.base.vue

```diff
@@ -93,14 +99,27 @@ export default {
   },
   computed: {
     isSearchActive() {
-      return this.value?.length;
+      return this.value?.length ?? false;
```

Comment:
could we rewrite this in terms of `isActiveSearchEmpty`? It seems the opposite thing. Could you do:
```javascript
isSearchActive() {
   return !this.isActiveSearchEmpty;
}
```

or 

```javascript
showDelete() {
      if (isNil(this.searchValue) && this.searchValue.length === 0) || !this. isSearchActive) return false;
      return true;
    },
```